fargate-services/2017-budget-api.yaml

```diff
@@ -73,11 +106,22 @@ Resources:
         Type: AWS::ECS::TaskDefinition
         Properties:
             Family: budget-service
+            Cpu: !Ref 'TaskCpu'
+            Memory: !Ref 'TaskMemory'
+            NetworkMode: awsvpc
+            RequiresCompatibilities:
+              - FARGATE
+            ExecutionRoleArn: !Ref ECSTaskExecutionRole
+            TaskRoleArn: !Ref TaskRole
             ContainerDefinitions:
                 - Name: budget-service
                   Essential: true
                   Image: 845828040396.dkr.ecr.us-west-2.amazonaws.com/integration/budget-service
-                  Memory: 100
+                  ## NOTE: this is probably duplicative of the Cpu/Memory defined just a few lines above...
+                  # CPU and Memory reservations are set to 50% of the default values specified above
+                  Cpu: 256
+                  Memory: 512
+                  MemoryReservation: 512
```

Comment:
Since we only run one container per task I would expect these values to be 100% of the provided TaskCpu and TaskMemory values.

Reply:
I would've expected CF to whine at me because these didn't add up either, but everything successfully deploys in this config, and I'd like to leave this as-is for now and better optimize resource allocations between staging and production in https://github.com/hackoregon/civic-devops/issues/246

Reply:
Follow-up: addressed this comment specifically with https://github.com/hackoregon/hackoregon-aws-infrastructure/pull/81